src/applications/search/containers/SearchApp.jsx

```diff
@@ -102,6 +102,21 @@ class SearchApp extends React.Component {
     if (queryChanged) {
       this.setState({ currentResultsQuery: userInput, page: 1 });
     }
+
+    recordEvent({
+      event: 'view_search_results',
+      'search-page-path': document.location.pathname,
+      'search-query': userInput,
+      'search-results-total-count': this.props.search?.totalEntries,
+      'search-results-total-pages': Math.ceil(
+        this.props.search?.totalEntries / 10,
```

Comment:
if for whatever reason totalEntries isn't there, this will result in `NaN`, maybe that's a good thing though since we'll pick it up in analytics and know there's an issue with it if it happens 🙂